app/models/APIAuthorisation.scala

```diff
@@ -14,31 +14,30 @@
  * limitations under the License.
  */
 
-package models
+package uk.gov.voa.businessrates.dataplatform.stub.models
 
+import models.{APIParty}
 import org.joda.time.{DateTime, LocalDate}
-import play.api.libs.functional.syntax._
-import play.api.libs.json._
+import play.api.libs.json.{Format, Json, Reads, Writes}
 
-case class APIAuthorisation(
-                             authorisationId: Long,
-                             uarn: Long,
-                             authorisationOwnerOrganisationId: Int,
-                             authorisationStatus: String,
-                             authorisationMethod: String,
-                             authorisationOwnerCapacity: String,
-                             createDatetime: DateTime,
-                             startDate: LocalDate,
-                             endDate: Option[LocalDate],
-                             submissionId: String,
-                             NDRListValuationHistoryItems: Seq[APIValuationHistory],
-                             parties: Seq[APIParty]
+case class APIAuthorisation(id: Option[Long],
+                            uarn: Long,
+                            authorisationOwnerOrganisationId: Long,
+                            authorisationOwnerPersonId: Option[Long],
+                            authorisationStatus: String,
+                            authorisationMethod: String,
+                            authorisationOwnerCapacity: String,
+                            createDatetime: DateTime,
+                            startDate: LocalDate,
+                            endDate: Option[LocalDate],
+                            submissionId: String,
+                            parties: Option[Seq[APIParty]]
```

Comment:
This is what comes back from the backend API, so we've got to parse it. It makes sense that on our side this is translated as just an empty seq at some point though